src/pages/ReportDetailsPage.js

```diff
@@ -142,12 +151,20 @@ class ReportDetailsPage extends Component {
                                             shouldUseFullTitle={isChatRoom || isPolicyExpenseChat}
                                         />
                                     </View>
-                                    <Text
-                                        style={[styles.sidebarLinkText, styles.optionAlternateText, styles.textLabelSupporting, styles.mb2, styles.pre]}
-                                        numberOfLines={1}
-                                    >
-                                        {chatRoomSubtitle}
-                                    </Text>
+                                    {isPolicyAdmin ? (
+                                        <Pressable
+                                            onPress={() => {
+                                                if (!isPolicyAdmin) {
```

Comment:
Out of curiosity, why do we need to check this condition again, given that we checked it 3 lines above?